lib/initial-setup/WP_Auth0_InitialSetup_Connections.php

```diff
@@ -3,239 +3,18 @@
-		),
-		array(
-			'provider' => 'vkontakte',
-			'name'     => 'vkontakte',
-			'icon'     => 'vk',
-		),
-		array(
-			'provider' => 'yandex',
-			'name'     => 'yandex',
-			'icon'     => 'Yandex Metrica',
-		),
-		array(
-			'provider' => 'thirtysevensignals',
-			'name'     => 'thirtysevensignals',
-			'icon'     => '37signals',
-		),
-		array(
-			'provider' => 'box',
-			'name'     => 'box',
-			'icon'     => 'Box',
-		),
-		array(
-			'provider' => 'salesforce',
-			'name'     => 'salesforce',
-			'icon'     => 'Salesforce',
-		),
-		array(
-			'provider' => 'salesforce-sandbox',
-			'name'     => 'salesforce-sandbox',
-			'icon'     => 'SalesforceSandbox',
-		),
-		array(
-			'provider' => 'salesforce-community',
-			'name'     => 'salesforce-community',
-			'icon'     => 'SalesforceCommunity',
-		),
-		array(
-			'provider' => 'fitbit',
-			'name'     => 'Fitbit',
-			'icon'     => 'Fitbit',
-		),
-		array(
-			'provider' => 'baidu',
-			'name'     => '百度 (Baidu)',
-			'icon'     => 'Baidu',
-		),
-		array(
-			'provider' => 'renren',
-			'name'     => '人人 (RenRen)',
-			'icon'     => 'RenRen',
-		),
-		array(
-			'provider' => 'weibo',
-			'name'     => '新浪微 (Weibo)',
-			'icon'     => 'Weibo',
-		),
-		array(
-			'provider' => 'shopify',
-			'name'     => 'Shopify',
-			'icon'     => 'Shopify',
-		),
-		array(
-			'provider' => 'dwolla',
-			'name'     => 'Dwolla',
-			'icon'     => 'dwolla',
-		),
-		array(
-			'provider' => 'miicard',
-			'name'     => 'miiCard',
-			'icon'     => 'miiCard',
-		),
-		array(
-			'provider' => 'wordpress',
-			'name'     => 'wordpress',
-			'icon'     => 'WordPress',
-		),
-		array(
-			'provider' => 'yammer',
-			'name'     => 'Yammer',
-			'icon'     => 'Yammer',
-		),
-		array(
-			'provider' => 'soundcloud',
-			'name'     => 'soundcloud',
-			'icon'     => 'Soundcloud',
-		),
-		array(
-			'provider' => 'instagram',
-			'name'     => 'instagram',
-			'icon'     => 'Instagram',
-		),
-		array(
-			'provider' => 'evernote',
-			'name'     => 'evernote',
-			'icon'     => 'Evernote',
-		),
-		array(
-			'provider' => 'evernote-sandbox',
-			'name'     => 'evernote-sandbox',
-			'icon'     => 'Evernote',
-		),
-		array(
-			'provider' => 'thecity',
-			'name'     => 'thecity',
-			'icon'     => 'The City',
-		),
-		array(
-			'provider' => 'thecity-sandbox',
-			'name'     => 'thecity-sandbox',
-			'icon'     => 'The City Sandbox',
-		),
-		array(
-			'provider' => 'planningcenter',
-			'name'     => 'planningcenter',
-			'icon'     => 'Planning Center',
-		),
-		array(
-			'provider' => 'exact',
-			'name'     => 'exact',
-			'icon'     => 'Exact',
-		),
-
-	);
 
 	public function __construct( WP_Auth0_Options $a0_options ) {
 		$this->a0_options = $a0_options;
-
-		add_action( 'wp_ajax_a0_initial_setup_set_connection', array( $this, 'update_connection' ) );
 	}
 
 	public function render( $step ) {
-		wp_enqueue_script( 'wpa0_async', WPA0_PLUGIN_LIB_URL . 'async.min.js' );
```

Comment:
Connection pieces were removed, other vars were moved to the `templates/initial-setup/connections.php` template below